dev/tests/static/testsuite/Magento/Test/Integrity/Dependency/DependencyProvider.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Test\Integrity\Dependency;
+
+use Magento\Framework\App\Utility\Files;
+use Magento\Framework\Component\ComponentRegistrar;
+
+class DependencyProvider
+{
+    /**
+     * Types of dependency between modules.
+     */
+    const TYPE_HARD = 'hard';
```

Comment:
looks like we have a new requirement to add "public" to const @see https://github.com/magento/magento2/pull/28713/files#diff-16beb87a670284151b84072a3fc8cc4dR17